app/assets/javascripts/experiments/table.js

```diff
@@ -196,7 +196,8 @@ var ExperimnetTable = {
       $('#modal-edit-module').data('id', this.selectedMyModules[0]);
       $('#edit-module-name-input').val($(`#taskName${$('#modal-edit-module').data('id')}`).data('full-name'));
     });
-    $('#modal-edit-module').on('click', 'button[data-action="confirm"]', () => {
+
+    const handleRenameModal = () => {
       let id = $('#modal-edit-module').data('id');
       let newValue = $('#edit-module-name-input').val();
 
```

Comment:
Actually, I just mimicked what was done here.
After closer inspection, it seems that the `this.clearRowTaskSelection()` was not working properly, because the CSS was missing a `,`.